### ISSUE TITLE
COBOL Parser error on PERFORM THRU statements with intermediate labels

### ISSUE THREAD

**Issue by codemanyak** [fesch/Structorizer.Desktop#1051]
The COBOL parser may fail during the postprocessing among the generated diagrams with an uncaught NullPointerException if there are further labels (internal procedure entry points) between the ones named by a PERFORM THRU statement (see #851), i.e. for the following case:
```cobol
       PERFORM foo THRU bar.
```

```cobol
       foo.
      *      some code
       inter1.
      *      more code
       inter2.
      *      even more code
       bar.
      *      final code (usually an EXIT)
```

_Originally posted by @codemanyak in https://github.com/fesch/Structorizer.Desktop/issues/1048#issuecomment-1198620368_

**Comment by codemanyak** [comment]
Bugfix is prepared, more instructive element comments will be generated from version 3.32-09 on, e.g.:
![grafik](https://user-images.githubusercontent.com/15326471/181652854-c2c0d9d6-04af-43cc-8bc4-0f139a7fa8c2.png)

**Comment by codemanyak** [comment]
After some tests with larger code samples it turned out to be utterly tedious for users to do this cleanup manually over several dozens of occurrences. On the one hand it can be very helpful for error detection to get this half-boiled result, on the other hand the tidying efforts are not reasonable if no severe mistakes occur.
A general cleanup mechanism in Structorizer for this COBOL-import-specific intermediary result wouldn't be reasonable either, so the best way seems to be to offer an automatic tidying step as additional import option for COBOL: By default the composed Calls would be split and cleared of void calls, and the no longer referenced empty subroutine diagrams (typically containing just an disabled exit element) could be deleted. In case of doubt, the import could be repeated without this tidying option chosen in order to investigate defective results.

**Comment by codemanyak** [comment]
New import option is implemented for version 3.32-09:
![grafik](https://user-images.githubusercontent.com/15326471/181777700-323fce66-5d42-43eb-8083-8bd9c217a9d9.png)

It offers three modes:
1. "tidy calls and routines": Complete tidying, i.e., combined calls will be parted, calls to expendable routines will be removed and the expnedable routines theselves as well.
2. "tidy calls": Like before, but the unproductive routines will not be removed from the group.
3. "don't tidy": No tidying is done, i.e., the multi-line calls remain as are, all COBOL paragraphs converted to routines and at least somewhere referred will remain. This mode is intended for better import trouble analysis.
